ipam/ipv6Ipam_test.go

```diff
@@ -36,59 +40,112 @@ func newKubernetesTestClient() kubernetes.Interface {
 	return client
 }
 
-func TestIPv6Ipam(t *testing.T) {
-	options := make(map[string]interface{})
-	options[common.OptEnvironment] = common.OptEnvironmentIPv6NodeIpam
-
-	client := newKubernetesTestClient()
-	node, _ := client.CoreV1().Nodes().Get(context.TODO(), testNodeName, metav1.GetOptions{})
-
-	testInterfaces, err := retrieveKubernetesPodIPs(node, testSubnetSize)
-	if err != nil {
-		t.Fatalf("Failed to carve addresses: %+v", err)
-	}
+func TestIpv6Ipam(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Ipv6Ipam Suite")
+}
 
-	correctInterfaces := &NetworkInterfaces{
-		Interfaces: []Interface{
-			{
-				IsPrimary: true,
-				IPSubnets: []IPSubnet{
-					{
-						Prefix: "ace:cab:deca:deed::/126",
-						IPAddresses: []IPAddress{
-							{Address: "ace:cab:deca:deed::2", IsPrimary: false},
-							{Address: "ace:cab:deca:deed::3", IsPrimary: false},
+var (
+	_ = Describe("Test ipv6Ipam", func() {
+
+		Describe("Test newIPv6IpamSource", func() {
+
+			Context("When using windows environment", func() {
+				It("Should create successfully", func() {
+					var kubeConfigPath string
+					name := "test"
+					options := map[string]interface{}{}
+					options[common.OptEnvironment] = name
+					if runtime.GOOS == windows {
+						kubeConfigPath = defaultWindowsKubeConfigFilePath
+					} else {
+						kubeConfigPath = defaultLinuxKubeConfigFilePath
+					}
```

Comment:
The Less, the tidy, and define `kubeConfigPath` when being used here.
```suggestion
					kubeConfigPath := defaultLinuxKubeConfigFilePath
					if runtime.GOOS == windows {
						kubeConfigPath = defaultWindowsKubeConfigFilePath
					} 
```

Reply:
done